client/src/components/AppComponent.jsx

```diff
@@ -89,6 +95,9 @@ export default class AppComponent extends React.Component {
                                 to="/java"
                                 value={0}
                             />
+                            <Tab
+                                label={localStorage.getItem('name')}
+                            />
```

Comment:
show in Navbar user's name (looks better from Usability point of view) . but of course if someone is against when delete it)